achievements/box.md

```diff
@@ -0,0 +1,45 @@
+# Box Achievements
+
+[Humbug Box][box] is an application bundler for PHP apps. It allows one to ship a PHP app into a [PHAR][phar] and distribute
+it with ease.
+
+Over the last few months, Box has been growing in popularity especially within the [Laravel][laravel] community thanks to the
+[Laravel Zero][laravel-zero] framework, a spin-off of Laravel more focused on CLI applications.
+
+In contrast, while Box is being used with a lot of projects making use of [Symfony][symfony] components, it cannot be used – at
+least easily, with a standard Symfony application.
+
+Thanks to this hackathon, [I][me] could, besides working on other Symfony related issues, and with the help of 
+[@nicolas-grekas][nicolas], [@stof][stof] and [@tucksaun][tucksaun], work on adding support for Symfony to Box.
```

Comment:
stof link does not display correctly (see rendered page), missing in the list below